Survey/Survey/Sources/Data/Models/LoginTokenApi.swift

```diff
@@ -36,7 +36,21 @@ struct LoginTokenApi: Codable, LoginToken {
         createdAt = try? attributes.decode(Date.self, forKey: .createdAt)
     }
 
-    func encode(to encoder: Encoder) throws {
+    public init(
+        accessToken: String,
+        tokenType: String,
+        expiresIn: Int,
+        refreshToken: String,
+        createdAt: Date?
+    ) {
+        self.accessToken = accessToken
+        self.tokenType = tokenType
+        self.expiresIn = expiresIn
+        self.refreshToken = refreshToken
+        self.createdAt = createdAt
+    }
+
+    public func encode(to encoder: Encoder) throws {
```

Comment:
These added `public`(s) seem redundant. Correct me if I miss anything here 🙏

Reply:
updated in this [commit](https://github.com/Shayokh144/ic-survey-swiftui-combine/pull/45/commits/f5457957a39d6c468022d4b12d157c568b988482)